src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -431,6 +451,73 @@ public MockResponse dispatch(RecordedRequest request) {
                     .isEqualTo(STUDY_ALREADY_EXISTS.name());
         }
 
+        // Create study with a bad case file -> error
+        try (InputStream is = new FileInputStream(ResourceUtils.getFile("classpath:" + TEST_FILE_WITH_ERRORS))) {
```

Comment:
Perhaps it would be better to create a separate test `testCreationWihtError`

Reply:
Done